src/Stitching/Stitching/Middleware/DelegateToRemoteSchemaMiddleware.cs

```diff
@@ -282,7 +283,12 @@ private static Path RewriteErrorPath(IError error, Path path)
 
             IStitchingContext stitchingContext = context.Service<IStitchingContext>();
             ISchema remoteSchema = stitchingContext.GetRemoteSchema(schemaName);
-            IComplexOutputType type = remoteSchema.GetOperationType(operationType);
+            IComplexOutputType type = remoteSchema.GetOperationType(OperationType.Mutation);
+            if(type is null)
+            {
+                type = remoteSchema.GetOperationType(OperationType.Query);
+            }
```

Comment:
we should get the operation type from the delegate directive here. how do I access that?